index.bs

```diff
@@ -455,6 +476,46 @@ dictionary XRSessionCreationOptions {
 };
 </pre>
 
+XRRenderState {#xrrenderstate-interface}
+-------------
+
+There are multiple values that developers can configure which affect how the session's output is composited. These values are tracked by an {{XRRenderState}} object.
+
+<pre class="idl">
+dictionary XRRenderStateInit {
+  double depthNear;
+  double depthFar;
+  XRLayer baseLayer;
+};
+
+[SecureContext, Exposed=Window] interface XRRenderState {
+  readonly attribute double depthNear;
+  readonly attribute double depthFar;
+  readonly attribute XRLayer? baseLayer;
+};
+</pre>
+
+<div class="algorithm" data-algorithm="initialize-renderstate">
+
+When an {{XRRenderState}} object is created, the user agent MUST <dfn>initialize the render state</dfn> by running the following steps:
+
+  1. Let |state| be the newly created {{XRRenderState}} object.
+  1. Initialize |state|'s {{XRRenderState/depthNear}} to <code>0.1</code>.
+  1. Initialize |state|'s {{XRRenderState/depthFar}} to <code>1000.0</code>.
+  1. Initialize |state|'s {{XRRenderState/baseLayer}} to <code>null</code>.
+
+</div>
+
+<div class="algorithm" data-algorithm="update-renderstate">
+
+When changes to an {{XRRenderState}} object |state| with a {{XRRenderStateInit}} dictionary |newState| are requested, the user agent MUST <dfn>update the render state</dfn> by running the following steps:
+
+  1. If |newState|'s {{XRRenderStateInit/depthNear}} value is set, set |state|'s {{XRRenderState/depthNear}} to |newState|'s {{XRRenderStateInit/depthNear}}.
+  1. If |newState|'s {{XRRenderStateInit/depthFar}} value is set, set |state|'s {{XRRenderState/depthFar}} to |newState|'s {{XRRenderStateInit/depthFar}}.
+  1. If |newState|'s {{XRRenderStateInit/baseLayer}} is set, set |state|'s {{XRRenderState/baseLayer}} to |newState|'s {{XRRenderStateInit/baseLayer}}.
```

Comment:
Just to be clear "is set" includes if it's set to null, yes?

Reply:
Yes, but I'm not clear on if there's a better way to communicate that? This verbiage is taken from another specs that had similar-ish behavior (though I'm embarrassed to admit at the moment that I don't recall which one?)